pom.xml

```diff
@@ -252,7 +265,7 @@
         </pluginRepository>
     </pluginRepositories>
 
-    <build>
+<build>
```

Comment:
Something weird happened to the indentation here.

Reply:
I blame the gremlins for that one.

Reply:
😄